README.md

```diff
@@ -12,6 +12,10 @@ The OCDS bid extension introduces a new, flexible, top-level section to each con
 
 This extension can also be used to publish data on quotes.
 
+## Legal Context
+
+In the European Union, this extension's fields correspond to [eForms BG-7 (Notice Result), BG-320 (Tender) and BG-310 (Contract)](https://docs.ted.europa.eu/eforms/latest/reference/business-terms/). See [OCDS for eForms](https://standard.open-contracting.org/profiles/eforms/latest/en/) for the correspondences to eForms fields.
```

Comment:
BG-7 contains 3 BT's all of which are discarded in OCDS and only 1 BT from BG-310 uses this extension